### DISCUSSION TITLE
使用wireguard功能遇到的两个问题

### DISCUSSION THREAD

**Discussion by liuyunyue** [Boos4721/openwrt#137] [Bugs/问题反馈]
ax6主路由开通wireguard，电脑win10连接在主路由下，并开通smb
手机通过ipv6连接主路由wireguard

问题1：访问外网只能访问ipv4，**不能访问外网ipv6，这是默认没有添加访问外网ipv6的路由吗？**

问题2：防火墙-自定义那里添加
iptables -t nat -A POSTROUTING -o br-lan -j MASQUERADE
ip6tables -t nat -A POSTROUTING -o br-lan -j MASQUERADE
规则后，可以访问局域网地址的ipv4地址和IPv6地址主机，手机通过wireguard连接到win10的smb。
发现从smb拖大点的视频到手机，文件传输可以完成，**反之，从手机往smb拖视频文件，会在传输过程中提示“失败   未能复制文件”，手机wireguard客户端MTU改到1280都还会复制失败。
wireguard连接在此过程中貌似没有中断**

这两个问题如何解决？小弟虚心请教。。。。

**Comment by liuyunyue** [comment]
小弟局域网中有台armbian设备也搭建了wireguard，同样手机连到这台设备的wireguard再到同一台win10的smb，文件双向复制都正常，访问外网ipv6也正常

**Comment by Boos4721** [comment]
https://github.com/Boos4721/openwrt/discussions/135

**Reply by liuyunyue** [reply]
#135  是tcp流量？wireguard是udp流量
 这个问题，好像我目前刷过的所有版本都有。。。不管5.10还是5.15
